modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/security/permission/wrapper/test/CTOnDemandUserPermissionCheckerWrapperTest.java

```diff
@@ -99,6 +113,8 @@ public void testHasPermission() throws Exception {
 	@Inject
 	private CTOnDemandUserTicketGenerator _ctOnDemandUserTicketGenerator;
 
+	private Group _group;
```

Comment:
@brookedalton just use @DeleteAfterTestRun instead of deleting manually

Reply:
@david-truong the group gets deleted but the Users_Groups entry does not, causing the same issue. I think it might be related to LPS-134942